057_numberOfDaysInAMonth.js

```diff
@@ -0,0 +1,13 @@
+// 57. Number of days in a given month of a given year
+const numberOfDays = (month, year) => {
+  if (((year % 4) === 0) && (month === 2)) {
```

Comment:
Logic issue: Number of days in 2 2023 is 31. What happens when it is February and not a leap year
Also correct leap year logic